assignment1/src/assignment1/SinglyLinkedList.java

```diff
@@ -16,34 +18,34 @@ public SinglyLinkedList(){
 
     // Access methods
     /**
-     * Returns number of elements in linked list
      * @return number of elements in the linked list
      */
     public int getSize(){
+    	// Tc = 1c
         return size;
     }
 
     /**
-     * Checks whether linked list is empty
      * @return boolean indicating whether list is empty
      */
     public boolean isEmpty(){
+    	// Tc = 1ct
```

Comment:
```suggestion
    	// Tc = 1c
```
I think you made a typo?